### ISSUE TITLE
Clarify how to retrieve prototoken data for alternate spawn/At arguments

### ISSUE THREAD

**Issue by salmelo** [trioderegion/warpgate#160]
Whenever I try to pass a `PrototypeToken` to `spawn` I get the following error:
```
caught (in promise) TypeError: undefined. Cannot read properties of null (reading 'update')
[Detected 1 package: warpgate]
    at _updateActor (mutator.js:280:36)
    at async api._spawnAt (api.js:585:7)
_updateActor	@	mutator.js:280
setTimeout (async)		
(anonymous)	@	module.js:369
wait	@	module.js:369
_updateActor	@	mutator.js:277
_spawnAt	@	api.js:585
await in _spawnAt (async)		
_spawn	@	api.js:483
await in _spawn (async)		
(anonymous)	@	VM1079:1
```

**Comment by trioderegion** [comment]
Looks like I may have missed something else with the actor delta changes...I'm sure I can get it fixed.

Does this prototoken have an assigned actor? And could I see an example script?

**Comment by salmelo** [comment]
I'm getting it from the actor, my minimal repo is just `warpgate.spawn(game.actors.content[0].prototypeToken)`. My original use case was spawning a token for an actor I just created, (specifically a clock in wicked ones.) Name based spawning will work fine on the same actor that breaks with prototokens. 

I did just remember, should've put this in the original report sorry, but it does spawn the token way up in the top left corner of the map before it errors out. Double clicking that token gives a warning "This Token references an Actor which no longer exists within the World."
